spec.md.html

```diff
@@ -120,16 +122,14 @@
  \nonumber\\
 E_{m,avg} &=& 1.0 + \alpha_{uv} (-0.113 + \alpha_{uv} (-1.8695 + \alpha_{uv} (2.2268 - 0.83397 \alpha_{uv})))
  \\
-F_{m,avg} &=& 0.047619 \rho_{s,r,90} + 0.952381 \rho_{s,r,0}
+F_{m,avg} &=& \frac{1}{21} \rho_{s,r,90} + \frac{20}{21} \rho_{s,r,0}
 \end{eqnarray}
 
 Note that we convert from anisotropic to isotropic roughness $\alpha_{uv}$ to simplify the function.
 
-Implementations may choose to replace the multiple scattering Fresnel part with the single scattering Fresnel $F(\theta)$ if concerned about performance.
-
 ### Diffuse Component
 
-The diffuse contribution of the material is modeled with a simple Lambertian BRDF. Based on the amount of specular contribution from the microfacet components, the diffuse contribution needs to be multiplied with a directional-dependent factor to ensure energy conservation. Following the approach of [#Kel01] and [#KC17], the factor is calculated from the missing energy after specular reflection. Instead of a lookup table, we use an analytical fit to save expensive texture lookups. For more information see Section [Energy Compensation: Diffuse-Specular].
+The diffuse contribution of the material is modeled with a simple Lambertian BRDF. Based on the amount of specular contribution from the microfacet components, the diffuse contribution needs to be multiplied with a directional-dependent factor to ensure energy conservation. Following the approach of [#Kel01] and [#KC17], the factor is calculated from the missing energy after specular reflection.
```

Comment:
> 
diffuse contribution needs to be multiplied with a directional-dependent factor to ensure energy conservation.
> 
Ensuring energy conservation was actually possible with a bit more simpler approaches but they were suffering from darkening artifacts at grazing angles for high roughness values. This approach now ensures that right amount of energy is preserved and therefore looks more realistic. 
So, maybe we can say 'energy preservation' there instead of 'energy conservation'.